app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -51,7 +51,8 @@ class CallNotificationManager @Inject constructor(private val context: Context)
     // Channels
     private fun createIncomingCallsNotificationChannel() {
         val audioAttributes = AudioAttributes.Builder()
-            .setContentType(AudioAttributes.CONTENT_TYPE_MUSIC)
+            .setContentType(AudioAttributes.CONTENT_TYPE_SONIFICATION)
+            .setUsage(AudioAttributes.USAGE_NOTIFICATION_RINGTONE)
```

Comment:
Just to make sure, it would be good to test this changes on Android 8.0~8.1 and also Android 12 at least as I remember me and Yamil were having troubles with the vibration not doing the correct loop consistently and also the ringing not being played correctly (not in a loop as well)